sdk/src/main/java/io/opentdf/platform/sdk/Manifest.java

```diff
@@ -215,6 +230,170 @@ public int hashCode() {
         }
     }
 
+    static public class Binding {
+        public String method;
+        public String signature;
+
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) return true;
+            if (o == null || getClass() != o.getClass()) return false;
+            Binding binding = (Binding) o;
+            return Objects.equals(method, binding.method) && Objects.equals(signature, binding.signature);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(method, signature);
+        }
+    }
+
+    static public class Assertion {
```

Comment:
I like the separation of the `AssertionConfig` and `Assertion` classes but I think we might be able to get almost the same thing by making the `sign`, `verify`, etc. methods private. I think that _maybe_ they should be unless we want to expose them to clients of the SDK. Not sure about that at all.